domains/zeto/internal/zeto/zeto.go

```diff
@@ -345,6 +371,8 @@ func (z *Zeto) registerEventSignatures(eventAbis abi.ABI) {
 			z.transferSignature = event.SolString()
 		case "UTXOTransferWithEncryptedValues":
 			z.transferWithEncSignature = event.SolString()
+		case "UTXOWithdraw":
+			z.withdrawSignature = event.SolString()
```

Comment:
Just noting because I had to go look at the Zeto source code... `deposit` just emits the existing `UTXOMint` event, so it doesn't get a new entry here.